frame/bridge/s2s/chain/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+authors     = ["Darwinia Network <hello@darwinia.network>"]
+description = "FRAME pallet to s2s chain"
+edition     = "2018"
+homepage    = "https://darwinia.network/"
+license     = "GPL-3.0"
+name        = "darwinia-s2s-chain"
```

Comment:
This is is not a substrate pallet?

Maybe we should move it to primitives if we need it.

Reply:
yes it's not suitable to be here.
Should be moved to primitives and runtime.